class-pinterest-for-woocommerce.php

```diff
@@ -463,7 +464,7 @@ public function maybe_update_plugin() {
 		}
 
 		/**
-		 * Load Localisation files.
+		 * Load Localization files.
```

Comment:
Should it be uppercase or lowercase?

```suggestion
		 * Load localization files.
```